src/controllers/TidalController.js

```diff
@@ -0,0 +1,25 @@
+var config = {
+  supports: {
+    playpause: true,
+    next: true,
+    previous: true
+  },
+  useLazyObserving: true,
+  playStateClass: 'play-controls__main-button--playing',
+  nextSelector: '.icon-Playback_next',
+  previousSelector: '.icon-Playback_previous'
+}
+
+config.artworkImageSelector = '.player__image-container__image';
+config.frameSelector = '.player__wrapper';
+config.playStateSelector = '.play-controls__main-button--playing';
+config.playSelector = '.play-controls__play';
+config.pauseSelector = '.play-controls__pause';
+config.titleSelector = '.player__heading';
+config.artistSelector = '.player__links';
```

Comment:
which one?  Yeah if you can clean this up great, and i'll go clean up the other one (or if you want to get it all into this pr that would work too, but not necessary to land)